README.md

```diff
@@ -16,13 +15,63 @@ And then execute:
 $ bundle
 ```
 
-Or install it yourself as:
-```bash
-$ gem install govuk_publishing_components
+### Mount the component guide
+
+Mount the component guide in your application:
+
+```ruby
+# config/routes.rb
+unless Rails.env.production?
+  mount GovukPublishingComponents::Engine, at: "/component-guide"
+end
+```
```

Comment:
Is this a change that once done is okay to commit to that application? If so, presumably over time eventually all apps will have this already. Some clarification/expansion needed here?